chainparams.go

```diff
@@ -10,10 +10,6 @@ import (
 	litecoinWire "github.com/ltcsuite/ltcd/wire"
 )
 
-// activeNetParams is a pointer to the parameters specific to the currently
```

Comment:
should we move this whole type out of the lnd package while we're at it?

Reply:
Going to do this in a followup PR, but could also do it here

Reply:
Going to do this in #4487